docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+
+In other words, a low priority job will be executed only if the private partition has resources to execute a job submitted to it. However, if a member of the private partition submits a job during this time, then the low priority job will be automatically terminated and re-queued (preempted). See {reF}`low-priority-preemption`.
+
+Jobs submitted to the `lowpriority` partition have the lowest priority of any class of jobs and are only considered after all other high priority jobs have been placed. Jobs submitted by private partition members to their own partition always have higher priority.
+::::
+
+::::{dropdown} 5. Is the `lowpriority` partition default?
+No, the default partition will still be the `short` partition. This ensures that by default, jobs are not subject to preemption.
+::::
+
+::::{dropdown} 6. How do I submit a job to the `lowpriority` partition?
+The `lowpriority` partition is a distinct partition that must be requested explicitly. SLURM accepts a comma-separated list of partitions. If a job is scheduled to run in the `lowpriority` partition, it may be preempted and re-queued. This does not happen in other partitions. The `lowpriority` partition is the only partition that can be preempted. See examples in using {ref}`low-priority`.
+::::
+
+::::{dropdown} 7. How can I ensure that my `lowpriority` partition job can run on nodes with sufficient resources?
+It is important to specify the required resources in the job script, including memory, CPUs and GPUs. For CPUs, this can include the number of cores and the micro-architecture and for GPUs this is typically handled by “class”. The GPU classes that are configured on the HPC system include P100, V100, T4, and A100. See our documentation on {ref}`slurm-guide-index` and {ref}`working-gpus` for additional information.
+::::
+
+::::{dropdown} 8. What is the downside of submitting a job to the `lowpriority` partition?
+If a job is submitted to the `lowpriority` partition and a high priority job comes through that requires resources currently occupied by the low priority job, then that low priority job will be stopped/suspended within 30s and re-queued. 
+::::
+
+::::{dropdown} 9. When should I NOT use the `lowpriority` partition to run my jobs?
+Jobs running on the `lowpriority` partition always carry the risk of being suspended before their wall time ends if a high priority job requests those resources while the low priority job is running. If you have checkpointing (see {ref}`checkpoint-jobs`) implemented in your workflow, such abrupt suspension of jobs would not be an issue, since your intermediate calculations/data are saved, and you can re-start your jobs from the point of their suspension. However, if you do not have checkpointing techniques implemented, and/or you expect your jobs to run for a while (and re-running them in the event of preemption means it is going to start all over again and take even longer to complete), it is not ideal to use the `lowpriority` partition. 
```

Comment:
```suggestion
Jobs running on the `lowpriority` partition always carry the risk of being suspended by a high priority job before their wall time ends. If you have checkpointing (see {ref}`checkpoint-jobs`) implemented in your workflow, such abrupt suspension of jobs would not be an issue, since your intermediate calculations/data are saved, and you can re-start your jobs from the point of their suspension. However, if you do not have checkpointing techniques implemented, and/or you expect your jobs will run for a long time, it is not ideal to use the `lowpriority` partition. 
```

Change link to: 'checkpointing implemented in your workflow'.